compiler/rustc_hir_typeck/src/pat.rs

```diff
@@ -2053,7 +2053,11 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                     .iter()
                     .map(|(_, name)| {
                         let field_name = name.to_string();
-                        if is_number(&field_name) { format!("{field_name}: _") } else { field_name }
+                        if is_number(&field_name) {
+                            format!("{}: _", field_name)
```

Comment:
Why did you change this? `{field_name}: _` is fine, right?

Reply:
I agree with you. There was no need for the change. I will revert it.

Reply:
[10b60eb](https://github.com/rust-lang/rust/pull/132025/commits/10b60eba9bc55449721db912fe75bab11de43c4e) fixes this